etc/db_schema.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Setup/Declaration/Schema/etc/schema.xsd">
+   <table name="crud" resource="default" engine="innodb" comment="Data Base">
+      <column xsi:type="smallint" name="id" padding="6" unsigned="false" nullable="false" identity="true" comment="ID" />
+      <column xsi:type="varchar" name="sku" nullable="false" length="25" comment="Sku" />
+      <column xsi:type="varchar" name="customerid" nullable="true" length="25" comment="C_Id" />
+      <column xsi:type="varchar" name="quoteid" nullable="true" length="25" comment="Q_Id" />
+      <column xsi:type="timestamp" name="createdat" default="CURRENT_TIMESTAMP" nullable="false" comment="Created_At" />
```

Comment:
@NandhiniKarthi field name should be created or created_at. Also please add updated field to update whenever record is updated.